src/utils/applyTheme.test.ts

```diff
@@ -115,13 +115,13 @@ const themeConfig: ThemeConfig = {
         label: "Large Border Radius",
         plugin: "borderRadius",
         type: "number",
-        default: `8px`,
+        default: 8,
```

Comment:
Hmm, I wonder if this should stay a string or not. If I were writing some default css I wouldn't think to put a naked number and then it gets automatically converted to pixels.

Reply:
Aaron said "For now, I meant for this to default to px. In the future, I could see us wanting to add the option for the dev or marketer to change this."

I think we will eventually want support for picking the unit. I agree this is a little weird so for now maybe we keep the number type represented as a string

Reply:
Did Aaron want this to be just a number? Maybe it just depends on what the input is actually rendered as. Is it some sort of number slider or is it just a normal input? If that latter we can go with string for now and update later.

Reply:
In the stories, there's not a number field in the UI mockup and in the example theme.config.ts it's just a number.

The sidebar renders the default Puck number field which is just a box with up and down arrows.

I'll switch to string, I think that's more useable/intuitive for now

Reply:
Sounds good.